.eleventy.js

```diff
@@ -0,0 +1,52 @@
+const drafts = [
+  'CG-FINAL',
+  'CR',
+  'ED',
+  'FCGS',
+  'PR',
+  'REC',
+  'WD',
+  'latest'
+];
+
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPassthroughCopy('.htaccess');
+  eleventyConfig.addPassthroughCopy('LICENSE.md');
+  eleventyConfig.addPassthroughCopy('benchmarks/**/*.{jsonld,nq,md}');
+  eleventyConfig.addPassthroughCopy('contexts/**/*.{htaccess,html,jsonld}');
+  eleventyConfig.addPassthroughCopy('contexts/{event,person,place,recipe,remote-context}');
+  eleventyConfig.addPassthroughCopy('examples/**/*.{html,ttl,txt,json}');
+  eleventyConfig.addPassthroughCopy('favicon.ico');
+  eleventyConfig.addPassthroughCopy('fonts');
+  eleventyConfig.addPassthroughCopy('images/**/*.{htaccess,png,svg,xcf}');
+  eleventyConfig.addPassthroughCopy('ns/**/*.{html,jsonld}');
+  eleventyConfig.addPassthroughCopy('playground/**/*.{css,php,js}');
+  eleventyConfig.addPassthroughCopy('presentations');
+  eleventyConfig.addPassthroughCopy('schemas/**/*.json');
+  eleventyConfig.addPassthroughCopy('site.css');
+  eleventyConfig.addPassthroughCopy('spec/LICENSE.md');
+  for(const draft of drafts) {
+    eleventyConfig.addPassthroughCopy(`spec/${draft}`);
+  }
+  eleventyConfig.addPassthroughCopy('static');
+  eleventyConfig.addPassthroughCopy('test-suite');
+  eleventyConfig.addPassthroughCopy('utils');
+  eleventyConfig.ignores.add('CONTRIBUTING.md');
+  eleventyConfig.ignores.add('LICENSE.md');
+  eleventyConfig.ignores.add('README.rst');
+  eleventyConfig.ignores.add('benchmarks/README.md');
+  eleventyConfig.ignores.add('contexts/person.html');
+  eleventyConfig.ignores.add('examples');
+  eleventyConfig.ignores.add('images/README.md');
+  eleventyConfig.ignores.add('minutes/**/*');
+  eleventyConfig.ignores.add('ns/json-ld.html');
+  eleventyConfig.ignores.add('playground/dev/README.md');
+  eleventyConfig.ignores.add('presentations');
+  eleventyConfig.ignores.add('scripts');
+  eleventyConfig.ignores.add('spec/tools');
+  eleventyConfig.ignores.add('spec/LICENSE.md');
```

Comment:
These ignores can be moved to `.eleventyignore` to make this file more sane.

Reply:
What's wrong with having them here?  At least for now.  It's a bit confusing as is due to eleventy processing various things, having to pass through others, and ignore particular files.  And for drafts, that's done computationally.  I thought it made more sense to keep the details in one place.

Reply:
I get the single place to look. I mostly just like data in inert places whenever possible vs. buried among API calls.